pkg/cmd/server/start/start_master.go

```diff
@@ -581,7 +592,9 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 
 		// called by admission control
 		kc.RunResourceQuotaManager()
-		oc.RunResourceQuotaManager(kc.ControllerManager)
+		if runOriginResourceQuotaController {
+			oc.RunResourceQuotaManager(kc.ControllerManager)
```

Comment:
Re-read the question and the answer is **NO**. It disables handling of just OpenShift-specific resource types.